### PR TITLE
Add a vanilla svelte app

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
The front end is to be rewritten in svelte.  There is some svelte boiler-plate code.  It seems preferable to add the code that has no meaning beyond "because this is svelte" in a separate step.

# Changes
Added a blank svelte app as described here: https://svelte.dev/blog/the-easiest-way-to-get-started#2_Use_degit
```
cd frontend
npx degit sveltejs/template svelte
cd svelte
node scripts/setupTypeScript.js
npm install
git add .
git commit
```

Note: This svelte code is not added into the canister as part of the build step.  It can be used as a base for local development, no more.

### CODE REVIEW

File: frontend/svelte/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import livereload from 'rollup-plugin-livereload';
+import { terser } from 'rollup-plugin-terser';
+import sveltePreprocess from 'svelte-preprocess';
+import typescript from '@rollup/plugin-typescript';
+import css from 'rollup-plugin-css-only';
+
+const production = !process.env.ROLLUP_WATCH;
+
+function serve() {
+	let server;
```
Comment (ielashi) [inline frontend/svelte/rollup.config.js:13]: nit: These files are using tabs. Should we switch them to spaces so that our whole codebase is consistent?

**Review by ielashi — APPROVED** [review]
LGTM.

File: frontend/svelte/rollup.config.js (hunk 1 above)
Comment (bitdivine) [reply frontend/svelte/rollup.config.js:13]: Yes, can do.  I'll add eslint and prettier in the next step.

Comment (bitdivine) [reply frontend/svelte/rollup.config.js:13]: Hmm, eslint is harder than I expected but prettier is enough for the tabs vs spaces so I'll proceed with that.

**Pull request merged**